samples/Directory.Build.props

```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\..\src\Datadog.Trace.ClrProfiler.Native\bin\$(Configuration)\$(Platform)\**"
+    <None Include="..\..\src\Datadog.Trace.ClrProfiler.Native\bin\$(Configuration)\$(Platform)\*.dll;..\..\src\Datadog.Trace.ClrProfiler.Native\bin\$(Configuration)\$(Platform)\*.so"
```

Comment:
Doesn't this disregard pdb files?

Reply:
Good catch! I added the pdb files in 2f28ef393c21750b20f37b8b9e6a1b799c527765.